src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java

```diff
@@ -41,7 +40,7 @@
  */
 public class JFromDateToDateChooser extends JPanel {
 	
-	private static final String DATE_TIME_FORMAT = "dd/MM/yyyy";
+	private static final String DATE_TIME_FORMAT = " dd/MM/yyyy";
```

Comment:
There was no other layout adjustment other than adding a space in the format?

Reply:
if I understand correctly the change...

Reply:
Yes; I added a space so when it figures out the preferred spacing for the field there is actually enough room to see the whole date.  I was trying to make the most simple change in the code and this is what I came up with :)

Reply:
what about setting the preferred size?
`dateFromDateChooser.setPreferredSize(new Dimension(90, 20));`

Reply:
I was going to do that but at the time I considered that a "bigger" change that you might not approve.  Besides as shown in another PR figuring out the exact size can be tricky.

Reply:
> as shown in another PR figuring out the exact size can be tricky

It depends by the layout manager involved, some honor preferredSize() others ignore it and eventually require different approaches to obtain the wanted result. For the FlowLayout it is ok to use it, although it should guess the size by the JComponent, that in this case is a compound one, that's why I think it's tricky fo it to guess.

Reply:
So what is the final decision?  Leave the code as in this PR or replace it with `setPreferredSize()`?

Reply:
I think it is safe to use `setPreferredSize()` in this case (although there could be a better way to make the FlowLayout manager to calculate the rendering size by its own)